packages/menu-button/src/index.tsx

```diff
@@ -145,7 +149,13 @@ export const Menu: React.FC<MenuProps> = ({ id, children }) => {
       // @ts-ignore
       window.__REACH_DISABLE_TOOLTIPS = true;
       window.requestAnimationFrame(() => {
-        focus(menuRef.current);
+        if (keepScrollPosition) {
```

Comment:
We need to include this in the dependency array.

Reply:
I added it. Thanks for pointing this out!